packages/utils/interactive-hooks/src/useAsPressable.ts

```diff
@@ -12,6 +12,7 @@ import { usePressability } from './usePressability';
 /**
  * hover specific state and callback helper
  */
+// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
My impression is this line undoes the ban that lets us cast stuff as `object`. I wonder if we should just remove it from our config given we use it so much.

Reply:
`object` is almost never what you want. `{}` generally isn't either, but ends up being safe when used as a prop type in react components (https://github.com/typescript-eslint/typescript-eslint/issues/2063#issuecomment-675156492)

Reply:
+1 to what @NickGerleman said.

I posted this above, but here it is again:
```
ERR! /~/packages/utils/test-tools/src/enzymeTests.ts
ERR!   12:10  error    Don't use `object` as a type. The `object` type is currently hard to use ([see this issue](https://github.com/microsoft/TypeScript/issues/21732)).
ERR! Consider using `Record<string, unknown>` instead, as it allows you to more easily inspect and use the keys  @typescript-eslint/ban-types
```
It's probably worth taking a look. I know too little of the code base to have an opinion.

Reply:
Should I take it then that for most things we cast to object, there's probably a more appropriate type that it could be cast to instead?

Reply:
Almost always. The proposed `Record<string, unknown>` is a good alternative if you don't know what the value type is. If you do know the value type, you should use that instead. You can try converting one at a time and see what happens. I wouldn't recommend bulk-converting all at the same time. That's a recipe for disaster 😛